easybuild/easyconfigs/n/NGS_DNA/NGS_DNA-3.4.2.eb

```diff
@@ -14,7 +14,7 @@ easyblock = 'Tarball'
 source_urls = [('http://github.com/molgenis/%s/archive/' % (name))]
 sources = [('%s.tar.gz' % (version))]
 
-checksums = ['c64c3846f5a5a0f0e8d26e5913924b19']
+checksums = ['c99ad818c970b755728c3aa983ae2bb6']
```

Comment:
Source zou dezelfde moeten zijn als voor NGS_DNA-3.4.2-bare, maar de checksum is anders. Één van beide kan niet kloppen... toch?

Reply:
klopt, die bare is outdated